### PR TITLE
PETALSSECAMEL-42 trim java-routes

### PR DESCRIPTION
**Pull request by glaplace**
Trim java-routes déclared in jbi.xml

throw InvalidJBIConfigurationException if className are null or empty (avoid NullPointerException if className are null)

https://jira.petalslink.com/browse/PETALSSECAMEL-42

### CODE REVIEW

File: petals-se-camel/src/test/java/org/ow2/petals/se/camel/utils/CamelRoutesHelperTest.java [hunk 1]
```diff
@@ -93,4 +93,17 @@ public void testLoadRoutesClass_ko_nonloadable() throws Exception {
                 Logger.getLogger("TEST"));
     }
 
+    @Test
+    public void testLoadRoutesClass_ko_empty_className() throws InvalidJBIConfigurationException {
+        thrown.expect(InvalidJBIConfigurationException.class);
+        thrown.expectMessage("is not a subclass of Camel RouteBuilder");
```
Comment (cdeneux) [inline petals-se-camel/src/test/java/org/ow2/petals/se/camel/utils/CamelRoutesHelperTest.java]: Should not the message be "className must not be null or empty" ?

File: petals-se-camel/src/test/java/org/ow2/petals/se/camel/utils/CamelRoutesHelperTest.java [hunk 2]
```diff
@@ -93,4 +93,17 @@ public void testLoadRoutesClass_ko_nonloadable() throws Exception {
...
+        CamelRoutesHelper.loadRoutesFromClass(getClass().getClassLoader(), "");
+    }
+
+    @Test
+    public void testLoadRoutesClass_ko_null_className() throws InvalidJBIConfigurationException {
+        thrown.expect(InvalidJBIConfigurationException.class);
+        thrown.expectMessage("is not a subclass of Camel RouteBuilder");
```
Comment (cdeneux) [inline petals-se-camel/src/test/java/org/ow2/petals/se/camel/utils/CamelRoutesHelperTest.java]: Should not the message be "className must not be null or empty" ?

**Review by cdeneux — CHANGES_REQUESTED** [review]

**Review by glaplace — COMMENTED** [review]
Correct Message in Unit tests

**Review by cdeneux — APPROVED** [review]

**Comment by cdeneux** [comment]
Thanks @glaplace for your contribution !!!

**Pull request merged**